internal/supervisor/installation_test.go

```diff
@@ -531,7 +535,7 @@ func TestInstallationSupervisor(t *testing.T) {
 	t.Run("creation requested, cluster installations not yet created, no empty clusters", func(t *testing.T) {
 		logger := testlib.MakeLogger(t)
 		sqlStore := store.MakeTestSQLStore(t, logger)
-		supervisor := supervisor.NewInstallationSupervisor(sqlStore, &mockInstallationProvisioner{}, &mockAWS{}, "instanceID", 80, 0, false, false, &utils.ResourceUtil{}, logger)
+		supervisor := supervisor.NewInstallationSupervisor(sqlStore, &mockInstallationProvisioner{}, &mockAWS{}, "instanceID", false, false, standardSchedlingOptions, &utils.ResourceUtil{}, logger)
```

Comment:
Did you mean to omit the `u` on this variable name? If you did, I kind of think it's not worth omitting it to save the length and the variable should just be `standardSchedulingOptions`.

Reply:
Nope, didn't mean to omit it and is just a typo. Thanks for catching it.